moshi/moshi/models/lm.py

```diff
@@ -545,7 +559,7 @@ def _init_streaming_state(self, batch_size: int) -> _LMGenState:
         lm_model = self.lm_model
         initial = lm_model._get_initial_token()
         cache = torch.full(
-            (batch_size, self.lm_model.num_codebooks, self.max_delay + 2),
+            (batch_size, self.lm_model.num_codebooks, self.max_delay + 1000),
```

Comment:
What's the rationale behind the `1000`? Could you add a comment to explain it?

Reply:
Left over from my debugging!